### PR TITLE
create mobile view for welcome page

### PR DESCRIPTION
**Pull request by leeannachen**
mobile view (iPhoneX):
image
image

regular view:
image

### CODE REVIEW

**Review by aokeke — APPROVED** [review]
LGTM

File: proj/src/main/webapp/static/welcome.html [hunk 1]
```diff
@@ -1,75 +1,78 @@
...
-    <title>Welcome</title>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+
+    <title>camaraderie</title>
+
+    <!-- Styling -->
     <!-- Bootstrap: Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <!-- Bootstrap: jQuery library -->
+    <!-- Bootstrap: jQuery library -->
```
Comment (showermat) [inline proj/src/main/webapp/static/welcome.html:11]: Doesn't really matter, but could you fix this extra whitespace?

**Review by showermat — APPROVED** [review]
Nice work!

File: proj/src/main/webapp/static/welcome.html (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/welcome.html:11]: ofc

**Review by arringtonh — COMMENTED** [review]
lgtm

**Pull request merged**